gulpfile.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-console */
+/* eslint-env node */
+
+'use strict';
+
+const { exec } = require('child_process');
+
+const { parallel, watch } = require('gulp');
+
+function build(cb) {
+  exec('make build', (error, stdout, stderr) => {
+    console.error(stderr);
+    console.log(stdout);
+    cb(error);
+  });
+}
+
+function watchClient() {
+  watch('node_modules/hypothesis', { events: 'all' }, build);
+}
```

Comment:
My typical workflow for updating the Hypothesis client involves using [yalc](https://github.com/whitecolor/yalc) to "publish" the client on my local system and then use it within the browser extension repo. Running `yalc link hypothesis` doesn't trigger a rebuild currently.

This isn't a workflow that we recommend in the project README so I think that issue is OK for now, but I'll probably take a look when I have time.

Reply:
I have not tried with `yalc`, I currently use `yarn link`. I can take a look at this on a follow up.